WordPress/src/test/java/org/wordpress/android/ui/jetpack/scan/ScanViewModelTest.kt

```diff
@@ -89,6 +93,18 @@ class ScanViewModelTest : BaseUnitTest() {
         )
     }
 
+    @Test
+    fun `given last scan state not present in db, when vm starts, then loading scan state is displayed`() = test {
+        whenever(scanStore.getScanStateForSite(site)).thenReturn(null)
+        val uiStates = init().uiStates
+
+        val state = uiStates.first() as FullScreenLoadingUiState
```

Comment:
🔍 Nitpick: It seems that this tests doesn't have just a single reason to fail. We might want to consider splitting it into two tests.
1. One that check that `uiStates.first()` is instance of `FullScreenLoadingUiState`
2. One that checks that content of `FullScreenLoadingUiState` is what we expect

Same applies to some of the other tests, wdyt?

Reply:
It confuses me always, split done in 273866b.